index.bs

```diff
@@ -1739,6 +1782,219 @@ contexts that have active documents; navigation can also cause contexts to
 become inaccessible but not yet get discarded because bfcache.
 </div>
 
+## Log ## {#module-log}
+
+The <dfn export for=modules>log</dfn> module contains functionality and events
+related to logging.
+
+### Definition ### {#module-log-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+Command = (GetCommand)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+
+LogEvent = (
+  LogEntryAddedEvent
+)
+</pre>
+
+### Types ### {#module-log-types}
+
+#### log.LogEntry #### {#types-log-logentry}
+
+<pre class="cddl local-cddl">
+
+LogLevel = "debug" / "info" / "warning" / "error"
+
+LogEntry = {
+  type: text,
+  level: LogLevel,
+  text: text / null,
```

Comment:
Log message can have an array of parameters. Should the BiDi hide it and scalar all of them into a single string?

Reply:
That's handled in `ConsoleLogExtra` for the console case. I assume things like js exceptions really are just strings (this was based on https://chromedevtools.github.io/devtools-protocol/tot/Log/#event-entryAdded )

Reply:
Got it, thanks.